SPDY/SPDYSessionManager.m

```diff
@@ -182,7 +188,7 @@ - (void)_fillSessionPool:(SPDYSessionPool *)sessionPool cellular:(bool)cellular
     SPDYConfiguration *configuration = [SPDYProtocol currentConfiguration];
     NSUInteger size = configuration.sessionPoolSize;
 
-    while (sessionPool.count < size) {
+    for (NSUInteger i = sessionPool.count; i < size; i++) {
```

Comment:
This was a major bug I discussed in this commit's message. I'll just paste it here. :) Basically, if we hit that "continue" line (204) just below, we could get into a near infinite loop since the pool size was never increasing.

    There was a major bug in _fillSessionPool. If the pool size was > 1, and
    if the first SPDYSession (socket) was successfully created but
    subsequent ones failed, it would enter an infinite loop until they were
    eventually successfully created. This is an unlikely, if not nearly
    impossible, scenario in the real world, as I cannot find a way to make
    the SPDYSocket fail to initialize the socket except by artificial means.
    All failures, even "airplane mode", come asynchronously during a stream
    callback.